apps/prairielearn/src/lib/code-caller/code-caller-native.ts

```diff
@@ -69,30 +69,29 @@ export interface ErrorData {
 
 export type CodeCallerError = Error & { data?: ErrorData };
 
-/**
```

Comment:
I wasn't sure if this was attached to `CODE_CALLER_NATIVE`. I assumed it wasn't.